loopdb/migration_04_updates_test.go

```diff
@@ -0,0 +1,92 @@
+package loopdb
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/coreos/bbolt"
+	"github.com/stretchr/testify/require"
+)
+
+// TestMigrationUpdates asserts that the swap updates migration is carried out
+// correctly.
+func TestMigrationUpdates(t *testing.T) {
```

Comment:
q: Should we also test migration failure? Would it be better if migration success/failure would be tested with actual dbs?

Reply:
It is tested with actual dbs? We've tried so many styles of migrations over the past year. Which specific variant do you have in mind exactly?

Reply:
What I meant is that we could have actual db files for all versions, and do the update tests on those in a sequence. Then test that the end result is key/value equivalent between steps/in the end.

Reply:
Ah, so start with a v1 database instead of a v3 in this case, to cover all migrations in one swoop? Only thing is then that we can't test additions after v1?